src/lib/cht-api.ts

```diff
@@ -134,6 +135,20 @@ export class ChtApi {
     return this.axiosInstance.post(url, deactivationPayload);
   };
 
+  countContactsUnderPlace = async (docId: string): Promise<number> => {
+    const url = `medic/_design/medic/_view/contacts_by_depth`;
+    console.log('axios.get', url);
```

Comment:
Why are we logging a constant? Do we wish to keep it for a while?

Reply:
I personally like seeing when the server makes requests to CHT, so all outbound requests from this cht-api library are logged. Fred has also indicated that he didn't like this pattern, but for now I'll stay consistent with the file and we can remove everywhere if we wish to remove.